src/log.action.ts

```diff
@@ -53,8 +53,8 @@ const getRecordTitle = (modifiedRecord, currentAdmin) => {
 export const createLogAction =
   ({
     onlyForPostMethod = false,
-    options = {},
-  }: CreateLogActionParams = {}): After<ActionResponse> =>
+    options,
```

Comment:
Why did you remove defaults here and below? 🤔

Reply:
reverted